tldw_Server_API/tests/test_stripe_metering.py

```
@@ -4,19 +4,38 @@
 
 import pytest
 
+from tldw_Server_API.app.services import stripe_metering_service as _sms_module
 from tldw_Server_API.app.services.stripe_metering_service import StripeMeteringService
 
+# Mock stripe module for environments where stripe is not installed
+_mock_stripe = MagicMock()
+_mock_stripe.api_key = None
+
+
+def _stripe_ok():
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_stripe_ok`

(ANN202)